src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -959,7 +964,18 @@ private async Task<DownloadedBuild> GatherDropForBuildAsync(Build build, string
                                                                                 List<string> errors,
                                                                                 StringBuilder downloadOutput)
         {
-            string packageContentUrl = $"https://pkgs.dev.azure.com/{feedAccount}/{feedVisibility}_apis/packaging/feeds/{feedName}/nuget/packages/{asset.Name}/versions/{asset.Version}/content";
+            string assetName = asset.Name;
+
+            // Some blobs get pushed as packages. This is an artifact of a one-off issue in core-sdk
```

Comment:
>a one-off issue in core-sdk [](start = 73, length = 27)

Core-sdk is where I first saw this. Ideally that's the only place where it happens....